File: 2LSC_JovanyMendez_06.cpp
#include <iostream>
using namespace std;

int main() {
    int Numero1, Numero2, Numero3, Producto, Suma;

    cout << "Ingrese el primer numero: ";
    cin >> Numero1;
    cout << "Ingrese el segundo numero: ";
    cin >> Numero2;
    cout << "Ingrese el tercer numero: ";
    cin >> Numero3;

    if (Numero1 > 0) {
        Producto = Numero1 * Numero2 * Numero3;
        cout << "El producto de los numeros es: " << Producto << endl;
    } else {
        Suma = Numero1 + Numero2 + Numero3;
        cout << "La suma de los numeros es: " << Suma << endl;
    }

    return 0;
}

File: 2LSC_JovanyMendez_06.py
def main():
    Numero1 = int(input("Ingrese el primer número: "))
    Numero2 = int(input("Ingrese el segundo número: "))
    Numero3 = int(input("Ingrese el tercer número: "))

    if Numero1 > 0:
        Producto = Numero1 * Numero2 * Numero3
        print("El producto de los números es:", Producto)
    else:
        Suma = Numero1 + Numero2 + Numero3
        print("La suma de los números es:", Suma)

if __name__ == "__main__":
    main()
